geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/CommandManager.java

```diff
@@ -112,12 +121,10 @@ private static void raiseExceptionIfEmpty(Set<Class<?>> foundClasses, String err
     }
 
     // Find by packages specified in the distribution config
-    if (this.cacheProperties != null) {
-      String cacheUserCmdPackages =
-          this.cacheProperties.getProperty(ConfigurationProperties.USER_COMMAND_PACKAGES);
-      if (cacheUserCmdPackages != null && !cacheUserCmdPackages.isEmpty()) {
-        userCommandSources.add(cacheUserCmdPackages);
-      }
+    String cacheUserCmdPackages =
```

Comment:
I am not sure how the cacheProperties are initialized, will there be a case "cacheUserCmdPackages" could be null.

Reply:
`cacheProperties` is initialized to empty Properties on line 75